neuralmonkey/decoders/transformer.py

```diff
@@ -322,7 +347,8 @@ def body(*args) -> LoopState:
 
                 # See train_logits definition
                 logits = tf.matmul(output_state, self.decoding_w)
```

Comment:
Matmul s `decoding_w` a if logits je tu už tak časo, že by to stálo z toho udělat funkci.

Reply:
Souhlasim, ale resil bych to mimo tenhle PR.